controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -308,6 +308,13 @@ type DefaultMachinePoolSpec struct {
 	// must be equal or multiple of the availability zones count.
 	// +optional
 	Autoscaling *expinfrav1.RosaMachinePoolAutoScaling `json:"autoscaling,omitempty"`
+
+	// VolumeSize set the disk volume size for the default workers machine pool in Gib. The default is 300 GiB.
+	// +kubebuilder:validation:Minimum=75
+	// +kubebuilder:validation:Maximum=16384
```

Comment:
just to confirm those max & min same as OCM ?

Reply:
Yep, I took those from https://docs.openshift.com/rosa/cli_reference/rosa_cli/rosa-manage-objects-cli.html#rosa-create-machinepool_rosa-managing-objects-cli